packages/sdks/core-js-sdk/src/sdk/flow/index.ts

```diff
@@ -31,6 +32,7 @@ const withFlow = (httpClient: HttpClient) => ({
           interactionId,
           version,
           componentsVersion,
```

Comment:
Why do we send it twice?

Reply:
I wanted to keep it for backward compatibility so i can remove it in phases. 
But perhaps better to deliver the backend first and only after it's in production to deliver the sdk

Reply:
You forgot to remove the flow version `version?: number,`

Reply:
removed.